File: rb/utility/make_context.rb
# VisitorFrequenciesLucerneCh SDK utility: make_context
require_relative '../core/context'
module VisitorFrequenciesLucerneChUtilities
  MakeContext = ->(ctxmap, basectx) {
    VisitorFrequenciesLucerneChContext.new(ctxmap, basectx)
  }
end

File: py/utility/make_context.py
# VisitorFrequenciesLucerneCh SDK utility: make_context

from core.context import VisitorFrequenciesLucerneChContext


def make_context_util(ctxmap, basectx):
    return VisitorFrequenciesLucerneChContext(ctxmap, basectx)
